system/inc/system_update.h

```diff
@@ -54,6 +54,12 @@ typedef enum {
     MODULE_INFO_JSON_INCLUDE_PLATFORM_ID = 0x0001
 } module_info_json_flags_t;
 
+typedef enum {
```

Comment:
I would move these down into the comms layer as part of the definition of the notify_update_done callback definition so this layer isn't dependent upon system.